packages/addon-info/src/components/Story.js

```diff
@@ -214,6 +216,24 @@ export default class Story extends React.Component {
     );
   }
 
+  _getComponentDescription() {
+    let retDiv = null;
+
+    if (Object.keys(window.STORYBOOK_REACT_CLASSES).length) {
```

Comment:
@ndelangen my linter gave syntax errors because it did not know where `STORYBOOK_REACT_CLASSES` came from. I understand that `STORYBOOK_REACT_CLASSES` is a global variable and that should work without the need of `window` at all. Sorry for any confusion I really only added it to get rid of the errors from my linter. I would be more than willing to do a commit that removes the need for `window`.